docs/contributing/standards.md

```diff
@@ -0,0 +1,69 @@
+---
+id: standards
+title: Contributing
+sidebar_label: Standards
+---
+
+# Standards
+
+The following standards are used by FAST-DNA contributors for consistency across this repository.
+
+## Core principles
+
+* Build for maintainability
+* Build for scalability
+* Build for performance
+
+## Naming Conventions
+
+* Avoid all abbreviations, except where common industry acronyms
+* Use names that are clear and intuitive
+* Use semantic names instead of presentational
+
+## Code commenting
+
+* For work to complete later use `// TODO <issue number>: <title>` and file new issue
+* For work considered a hack use `// HACK <issue number>: <title>` and file new issue
+* Be verbose and do explain what you're doing and why you're doing it
+* Use `//` for single line comment blocks
```

Comment:
Jane and I both commented on the use of comments - please see your other PR for details.

Reply:
completely removed this section already